dashboard/app/models/peer_review.rb

```diff
@@ -52,6 +54,11 @@ def add_status_to_audit_trail
     append_audit_trail "REVIEWED by user id #{reviewer_id} as #{status}"
   end
 
+  after_save :send_review_completed_mail, if: :status_changed?
+  def send_review_completed_mail
+    PeerReviewMailer.review_completed_receipt(self).deliver_now
```

Comment:
Don't send email if the review has been escalated.

Reply:
done - now only send when it changes and is either accepted or rejected